modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/ColumnParams.java

```diff
@@ -169,7 +169,7 @@ public Builder precision(int precision) {
          *
          * @return {@code this}.
          */
-        public Builder scale(int scale) {
+        public Builder scale(Integer scale) {
```

Comment:
Please add `@Nullable` for method `org.apache.ignite.internal.catalog.commands.ColumnParams#scale` and specify in documentation what `null` means.